Aztec/Classes/Public/TextKit/TextAttachment.swift

```diff
@@ -1,20 +1,22 @@
 import Foundation
 import UIKit
 
-public protocol TextAttachmentImageProvider {
-    func image(forURL url: NSURL, forAttachment attachment: TextAttachment, onSuccess success: (UIImage) -> (), onFailure failure: () -> ()) -> UIImage?
+protocol TextAttachmentImageProvider {
+    func textAttachment(textAttachment: TextAttachment, imageForURL url: NSURL, onSuccess success: (UIImage) -> (), onFailure failure: () -> ()) -> UIImage
```

Comment:
Even though we don't have a specific rule in our style guide, [it does mention that RayWenderlich's guide is a good starting point for reference](https://github.com/wordpress-mobile/swift-style-guide).

You can see in [this issue](https://github.com/raywenderlich/swift-style-guide/issues/175) that they're already working towards adding this convention to their docs.  Here's [one of the commits](https://github.com/fabienwarniez/swift-style-guide/commit/25ccd3935c20c0954ff97d1688a66386aa5f6191) showing some clear examples.

Additionally, there's no indication that this only pertains to Objective-C code in Apple's delegates (UITableViewDelegate, UIPickerViewDelegate, etc).

All in all, going in this direction would basically ensure we'd be following a widely used convention.